.github/workflows/slsa-goreleaser.yml

```diff
@@ -7,9 +7,6 @@ on:
 
 permissions: read-all
 
-env:
-  GO_VERSION: 1.24
-
```

Comment:
Please revert this unrelated change

Reply:
Reverted.

This was to fix this:

<img width="917" height="268" alt="image" src="https://github.com/user-attachments/assets/31ace9d5-6408-4595-bb06-a60e8a3aecec" />